.gitignore

```diff
@@ -6,8 +6,13 @@
 pyenv/
 pyenv2/
 
+# tool-generated folders
+prof/
+IATI-Data-Snapshot/
+
 # defined files to ignore
 ignore_*
+test.py
```

Comment:
Not sure what this is for?

Reply:
https://github.com/IATI/iati.core/pull/142/commits/0881dc3c0ae70477752e7be527c37d7539332907